databricks-iris/{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -3,5 +3,5 @@ jupyterlab>=3.0
 notebook
 kedro~={{ cookiecutter.kedro_version }}
 kedro[jupyter]
-kedro-datasets[spark, pandas, spark.SparkDataset, pandas.ParquetDataset]>=1.0
+kedro-datasets[spark, pandas, spark.SparkDataset, pandas.ParquetDataset]>=3.0
```

Comment:
You'll need to update this to match with the 3.0 syntax e.g. `pandas-parquetdataset` instead of `pandas.ParquetDataset`